fastlane/Fastfile.swift

```diff
@@ -68,4 +68,16 @@ class Fastfile: LaneFile {
         Symbol.downloadFromAppStore(versionNumber: versionNumber, buildNumber: buildNumber)
         Symbol.uploadAppStoreToCrashlytics(versionNumber: versionNumber, buildNumber: buildNumber)
     }
+
+    // MARK: - Utilities
+
+    func cleanUpDerivedData() {
```

Comment:
I think we can have only one clean function to clear all things in `Output` folder.

Reply:
Correct! I updated at https://github.com/nimblehq/ios-templates/pull/366/commits/e8eef877f1d68b1bc3f281aa0b3077a7d199178e